src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -194,4 +252,25 @@ class KeyPhraseExtractionSuiteV4 extends TestBase with DataFrameEquality with Te
         row.toSeq.foreach { col => println(col) }
       }
     }
+
+  lazy val df3: DataFrame = Seq(
+    ("en","Hello world. This is some input text that I love."),
+    ("es", "La carretera estaba atascada. Había mucho tráfico el día de ayer."),
+    ("fr", "Bonjour tout le monde")
+  ).toDF("lang","text" )
+
+  test("Keyphrase - batch usage"){
+    lazy val extractor2: TextAnalyticsKeyphraseExtraction = new TextAnalyticsKeyphraseExtraction(options)
+      .setSubscriptionKey(textKey)
+      .setEndpoint("https://eastus.api.cognitive.microsoft.com/")
+      .setInputCol("text")
+      .setLangCol("lang")
+      .setOutputCol("output2")
+
+    val tdf = extractor2.transform(df3)
+      .select("output2.result.keyPhrases")
+      .collect()
+    assert(tdf.length == 1)

```

Comment:
What is this assertion checking here?

Reply:
it's checking that it is only batching into one row (expected because default is 5 and this is 3 inputs)